CMakeLists.txt

```diff
@@ -327,7 +342,11 @@ else() # ANDROID/UNIX
 	endif()
 
 	set(so_file $<TARGET_FILE:godot-jolt>)
-	set(debug_file ${so_file}.debug)
+	if(EMSCRIPTEN)
```

Comment:
Does it maybe make more sense to just have Emscripten have its own `elseif(EMSCRIPTEN)` in the top-level if-statement here? It seems like there are enough subtle differences to warrant it.

Reply:
I almost did it that way but got lazy. Your nudge confirms it was the correct choice. :grinning: Will fix!